managing-owners-using-ui.md

```diff
@@ -36,6 +38,26 @@ On confirmation, all the existing owners will be notified about the owner additi
 
 `Note that` the user won't have access to the gem until they confirm the ownership addition.
 
+Owner & Maintainer Roles
+------------------------
+When managing owners, you have the option to select a role, either Owner or Maintainer. Owners have full control over the gem, including the ability to add or remove owners. Maintainers, have the ability to publish and yank gem versions, but cannot manager users, or configure gem security settings.
```

Comment:
Typo: "cannot manager users"

Reply:
Fixed this in https://github.com/rubygems/guides/commit/c2c8a9f972a4bf4b2e0d56b9c632ef7f587e7207